index.html

```diff
@@ -0,0 +1,981 @@
+                  <polygon
+                    points="10.9,-69 5.2,-69 5.2,-68.1 11.4,-68.1 11.4,-69"
+                  ></polygon>
+                  <rect x="5.2" y="-67.3" width="6.1" height="0.9"></rect>
+                  <rect x="5.2" y="-65.5" width="6.1" height="0.9"></rect>
+                  <path
+                    d="M12,6.5H6.5V12H1V6.5h5.5V1H12V6.5zM12,0H1C0.4,0,0,0.5,0,1v11c0,0.6,0.4,1,1,1h11c0.5,0,1-0.4,1-1V1C13,0.5,12.5,0,12,0"
+                  ></path>
+                </svg>
+                Games</strong
+              ></a
+            >
+            <a href="">
+              <svg class="icons" viewBox="0 0 13 13" fill="#000">
+                <path
+                  d="M12,2.9L9.6,5.2c-0.1,0.1-0.3,0.1-0.4,0C9.1,5.2,9.1,5,9.3,4.9l2.4-2.4c-0.2-0.2-0.3-0.3-0.5-0.5L8.7,4.3c-0.1,0.1-0.3,0.1-0.4,0C8.2,4.3,8.2,4.1,8.4,4l2.4-2.4c-0.3-0.3-0.5-0.5-0.5-0.5L7.6,3.4C7.1,4,6.8,5.1,7.1,5.8c-1.4,1-4.6,3.5-5.1,4c-0.8,0.8-0.4,1.8-0.3,1.9c0,0,0,0,0,0c0,0,0,0,0,0c0.1,0.1,1.1,0.5,1.9-0.3c0.4-0.4,2.9-3.6,3.9-5C8.4,6.9,9.6,6.6,10.2,6l2.3-2.6C12.5,3.4,12.3,3.2,12,2.9z"
+                ></path>
+                <path
+                  d="M0.8,1.9l0.3-0.3c0.9-0.9,3.2,1.1,3.8,1.7s0.9,1.8,0.4,2.6c1.4,1.1,4.6,3.5,5,3.9c0.8,0.8,0.4,1.8,0.3,1.9c0,0,0,0,0,0c0,0,0,0,0,0c-0.1,0.1-1.1,0.5-1.9-0.3c-0.4-0.4-2.9-3.7-4-5.1C3.9,6.7,2.9,6.4,2.3,5.8S-0.2,2.9,0.8,1.9z"
+                ></path>
+              </svg>
+              <strong>Cooking</strong></a
+            >
+            <a href="" class="small-txt">Email Newsletters</a>
+            <a href="" class="small-txt">Corporate Subscriptions</a>
+            <a href="" class="small-txt">Education Rate</a>
+            <div class="divider-footer-subscribe-col"></div>
+            <a href="" class="small-txt">Mobile Applications</a>
+            <a href="" class="small-txt">Replica Edition</a>
+            <a href="" class="small-txt">International</a>
+            <a href="" class="small-txt">Canada</a>
+            <a href="" class="small-txt">Español</a>
+            <a href="" class="small-txt">中文网</a>
+          </div>
+        </div>
+        <nav class="footer-nav">
+          <a href="">© 2021 The New York Times Company</a>
+          <a href="">NYTCo</a>
+          <a href="">Contact Us</a>
+          <a href="">Accessibility</a>
+          <a href="">Work with us</a>
+          <a href="">Advertise</a>
+          <a href="">T Brand Studio</a>
+          <a href="">Your Ad Choices</a>
+          <a href="">Privacy Policy</a>
+          <a href="">Terms of Service</a>
+          <a href="">Terms of Sale</a>
+          <a href="">Site Map</a>
+          <a href="">Help</a>
+          <a href="">Subscriptions</a>
+        </nav>
+      </div>
+    </footer>
+    <div class="ad">
+      <div class="subscription">
+        <strong>Subscribe for $0.25 a week.</strong> &ThinSpace; Ends soon.
+      </div>
+      <div class="expand-btn">Expand</div>
+    </div>
+  </body>
+</html>
```

Comment:
### Optional Change
- [ ] Do you mind validating the markup? We have some informing warnings. Please feel free to use this [tool](https://validator.w3.org/nu/#textarea).